blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Context schema and export to CodeSandbox or StackBlitz
+description: You can now open your machine in CodeSandbox or StackBlitz. These options show when you select either JavaScript or TypeScript export formats in the Code panel.
+authors: [laura]
+tags: [changelog, new, improved, fixed]
+date: 2023-09-21
+slug: 2023-09-21-context-schema-and-export-to-external-editors
+image: /blog/2023-09-21-context-schema-and-export-to-external-editors.png
+---
+
+You can now open your machine in CodeSandbox or StackBlitz. These options show when you select either JavaScript or TypeScript export formats in the **Code** panel. When you open your machine in these external editors, it will be deployed as a demo app so you can jump quickly into prototyping and get an example of how to integrate your machine into your codebase.
+
+![Code panel where JavaScript is selected from the dropdown. There are toggles for XState V5 beta, Show descriptions, and Show meta. There are options to Copy code, Open in CodeSandbox, and Open in StackBlitz.](./export-to-external-editors.png)
+
+<!--truncate-->
+
+## Context schema
+
+We’ve now released the context schema in the editor. You can find the context schema above the event schemas in the machine **Details** panel. Here, you can add a new context property with its name, type (string, number, boolean, array and object), and an optional description. These will then be exported in your TypeScript code. When you import a machine, Stately will stub an empty schema for any context properties in your code. If you add a context property in the **Context** panel that’s missing from the context schema, a quick link will appear under that property to quickly add it to the schema.
+
+![The Context schema section at the end of the Machine details panel. The schema contains properties for timers and volume.](./context-schema.png)
+
+You can edit any context properties in the modal that opens when you click the property in the schema list and delete all of them from the **Delete all** option under the **...** context menu.
```

Comment:
```suggestion
You can edit any context properties in the modal that opens when you click the property in the schema list. To delete all schema properties, use the **Delete all** option under the **...** menu.
```

I'm suggesting to split editing and delete all into two separate sentences.
You could keep the "**...** context menu" but I found that in this context, that "context" is confused with the **Context** feature being described.